src/MongoDB.Bson.Tests/Specifications/bson/tests/decimal128-4.json

```diff
@@ -0,0 +1,165 @@
+          "match_string": "0.1000000000000000000000000000000000"
+       }
+    ],
+    "parseErrors": [
+       {
+          "description": " [basx564] Near-specials (Conversion_syntax)",
+          "subject": "Infi"
+       },
+       {
+          "description": " [basx565] Near-specials (Conversion_syntax)",
+          "subject": "Infin"
+       },
+       {
+          "description": " [basx566] Near-specials (Conversion_syntax)",
+          "subject": "Infini"
+       },
+       {
+          "description": " [basx567] Near-specials (Conversion_syntax)",
+          "subject": "Infinit"
+       },
+       {
+          "description": " [basx568] Near-specials (Conversion_syntax)",
+          "subject": "-Infinit"
+       },
+       {
+          "description": " [basx590] some baddies with dots and Es and dots and specials (Conversion_syntax)",
+          "subject": ".Infinity"
+       },
+       {
+          "description": " [basx562] Near-specials (Conversion_syntax)",
+          "subject": "NaNq"
+       },
+       {
+          "description": " [basx563] Near-specials (Conversion_syntax)",
+          "subject": "NaNs"
+       },
+       {
+          "description": " [dqbas939] overflow results at different rounding modes (Overflow & Inexact & Rounded)",
+          "subject": "-7e10000"
+       },
+       {
+          "description": " [dqbsr534] negatives (Rounded & Inexact)",
+          "subject": "-1.11111111111111111111111111111234650"
+       },
+       {
+          "description": " [dqbsr535] negatives (Rounded & Inexact)",
+          "subject": "-1.11111111111111111111111111111234551"
+       },
+       {
+          "description": " [dqbsr533] negatives (Rounded & Inexact)",
+          "subject": "-1.11111111111111111111111111111234550"
+       },
+       {
+          "description": " [dqbsr532] negatives (Rounded & Inexact)",
+          "subject": "-1.11111111111111111111111111111234549"
+       },
+       {
+          "description": " [dqbsr432] check rounding modes heeded (Rounded & Inexact)",
+          "subject": "1.11111111111111111111111111111234549"
+       },
+       {
+          "description": " [dqbsr433] check rounding modes heeded (Rounded & Inexact)",
+          "subject": "1.11111111111111111111111111111234550"
+       },
+       {
+          "description": " [dqbsr435] check rounding modes heeded (Rounded & Inexact)",
+          "subject": "1.11111111111111111111111111111234551"
+       },
+       {
+          "description": " [dqbsr434] check rounding modes heeded (Rounded & Inexact)",
+          "subject": "1.11111111111111111111111111111234650"
+       },
+       {
+          "description": " [dqbas938] overflow results at different rounding modes (Overflow & Inexact & Rounded)",
+          "subject": "7e10000"
+       },
+       {
+          "description": "Inexct rounding#1",
+          "string": "100000000000000000000000000000000000000000000000000000000001",
```

Comment:
damn you test format!
The BSON document is called "subject", while the textual representation is called "string" --- except for parseErrors *sigh*.

Thanks, fixed!